vnext/Microsoft.ReactNative.ProjectReunion/Microsoft.ReactNative.ProjectReunion.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
```

Comment:
do we need to lock the windows version? would `net5.0` suffice?

Reply:
we need a ".net framework built for windows sdk" moniker per cswinrt docs. In this case, this could probably be 17763